pkg/monitor/kubernetes_metric.go

```diff
@@ -63,4 +106,29 @@ func (km *KubernetesMetric) GetJobSequenceMetrics(metricName, jobID string, star
 		return nil, err
 	}
 	return result, nil
-}
\ No newline at end of file
+}
+
+func getQuerySqlByMetric(metricName, podNames string) string {
+	switch metricName {
```

Comment:
It might be better to use a QuerySQL map？

Reply:
value may be changed, QuerySQL map may find the right key and modify the value